qiime2/sdk/plugin_manager.py

```diff
@@ -105,25 +104,49 @@ def _integrate_plugin(self, plugin):
     # something wrong, the entire plugin fails to load any piece, like a
     # databases rollback/commit
 
-    @property
-    def importable_formats(self):
-        """Return formats that are importable.
+    def get_formats(self, *, include_all=False, plugin=None):
+        # Initialize formats dictionary
```

Comment:
This is certainly a subjective request, but I don't think that this, or any of the other comments in this PR add value to the source code, they all basically just say the same thing as the code itself. Please remove or revise. :light_rail: